artemis/reporting/export/main.py

```diff
@@ -0,0 +1,125 @@
+)
+
+
+def _build_message_template_and_print_path(date_str: str) -> Template:
+    output_message_template_file_name = os.path.join(OUTPUT_LOCATION, "message_template_" + date_str + ".jinja2")
+
+    message_template_content = build_message_template()
+    message_template = environment.from_string(message_template_content)
+
+    with open(output_message_template_file_name, "w") as f:
+        f.write(message_template_content)
+
+    print(f"Message template written to file: {output_message_template_file_name}")
+    return message_template
+
+
+def _install_translations_and_print_path(language: Language, date_str: str) -> None:
+    translations_file_name = os.path.join(OUTPUT_LOCATION, "translations_" + date_str + ".po")
+    compiled_translations_file_name = os.path.join(OUTPUT_LOCATION, "compiled_translations_" + date_str + ".mo")
+    install_translations(language, environment, translations_file_name, compiled_translations_file_name)
+
+    print(f"Translations written to file: {translations_file_name}")
+    print(f"Compiled translations written to file: {compiled_translations_file_name}")
+
+
+def _dump_export_data_and_print_path(export_data: ExportData, date_str: str) -> None:
+    output_json_file_name = os.path.join(OUTPUT_LOCATION, "output_" + date_str + ".json")
+
+    with open(output_json_file_name, "w") as f:
+        json.dump(export_data, f, indent=4, cls=JSONEncoderAdditionalTypes)
+
+    print(f"JSON written to file: {output_json_file_name}")
+
+
+def _build_messages_and_print_path(message_template: Template, export_data: ExportData, date_str: str) -> None:
+    output_messages_directory_name = os.path.join(OUTPUT_LOCATION, "messages_" + date_str)
+    # We dump and reload the message data to/from JSON before rendering in order to make sure the template
+    # will receive precisely the same type of objects (e.g. str instead of datetime) as the downstream tasks
+    # that will work with the JSON and the templates. This allows us to catch some bugs earlier.
+    export_data_dict = json.loads(json.dumps(dataclasses.asdict(export_data), cls=JSONEncoderAdditionalTypes))
+
+    os.mkdir(output_messages_directory_name)
+    for top_level_target in export_data_dict["messages"].keys():
+        with open(os.path.join(output_messages_directory_name, top_level_target + ".html"), "w") as f:
+            f.write(message_template.render({"data": export_data_dict["messages"][top_level_target]}))
+    print()
+    print(termcolor.colored(f"Messages written to: {output_messages_directory_name}", attrs=["bold"]))
+
+
+def main(
+    already_exported_reports_directory: Path,
+    tag: Optional[str] = typer.Option(None),
+    language: Language = typer.Option(Language.en_US.value),
+    custom_template_arguments: str = typer.Option(
+        "", help="Custom template arguments in the form of name1=value1,name2=value2,..."
+    ),
+    blocklist_file: Optional[str] = typer.Option(None),
+    verbose: bool = typer.Option(False, "--verbose"),
```

Comment:
Would be nice to have docs inline `-h` instead of going to website. It's already using typer, so it's not far ;)